databricks_cli/unity_catalog/table_cli.py

```diff
@@ -0,0 +1,170 @@
+              help='Name of the parent catalog for tables of interest.')
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def list_table_summaries_cli(api_client, catalog_name):
+    """
+    List table summaries (in bulk).
+
+    Calls the 'listTableSummaries' RPC endpoint of the Unity Catalog service.
+    Returns array of TableSummarys.
+
+    """
+    tables_json = UnityCatalogApi(api_client).list_table_summaries(catalog_name)
+    click.echo(mc_pretty_format(tables_json))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Get a table.')
+@click.option('--full-name', required=True,
+              help='Full name (<catalog>.<schema>.<table>) of the table to get.')
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def get_table_cli(api_client, full_name):
+    """
+    Get a table.
+
+    Calls the 'getTable' RPC endpoint of the Unity Catalog service.
+    Returns nothing.
+
+    """
+    table_json = UnityCatalogApi(api_client).get_table(full_name)
+    click.echo(mc_pretty_format(table_json))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Update a table.')
+@click.option('--full-name', required=True,
+              help='Full name (<catalog>.<schema>.<table>) of the table to update.')
+@click.option('--json-file', default=None, type=click.Path(),
+              help='File containing JSON request to PATCH.')
+@click.option('--json', default=None, type=JsonClickType(),
+              help=JsonClickType.help('/api/2.0/tables'))
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def update_table_cli(api_client, full_name, json_file, json):
+    """
+    Update a table.
+
+    Calls the 'updateTable' RPC endpoint of the Unity Catalog service.
+    Returns nothing.
+
+    """
+    json_cli_base(json_file, json,
+                  lambda json: UnityCatalogApi(api_client).update_table(full_name, json),
+                  encode_utf8=True)
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Delete a table.')
+@click.option('--full-name', required=True,
+              help='Full name (<catalog>.<schema>.<table>) of the table to delete.')
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def delete_table_cli(api_client, full_name):
+    """
+    Delete a table.
+
+    Calls the 'deleteTable' RPC endpoint of the Unity Catalog service.
+    Returns nothing.
+
+    """
+    UnityCatalogApi(api_client).delete_table(full_name)
+
+
+def register_table_commands(cmd_group):
+    cmd_group.add_command(create_table_cli, name='create-table')
+    cmd_group.add_command(list_tables_cli, name='list-tables')
+    cmd_group.add_command(list_table_summaries_cli, name='list-table-summaries')
+    cmd_group.add_command(get_table_cli, name='get-table')
+    cmd_group.add_command(update_table_cli, name='update-table')
+    cmd_group.add_command(delete_table_cli, name='delete-table')
```

Comment:
please make command naming consistent with the rest of the CLI (see comments for other command groups)

Reply:
Addressed in #471.